composer.json

```
@@ -1,6 +1,6 @@
 {
     "name": "laminas/laminas-psr7bridge",
-    "description": "PSR-7 &lt;-&gt; laminas-http message conversions",
+    "description": "PSR-7 to laminas-http message conversions",
```

```suggestion
    "description": "Bidirectional conversions between PSR-7 and laminas-http messages",
```

It's bi-directional, which is why we had `<->` previously. I'm suggesting something that makes that clear.

---

The new description must also be added to:

https://github.com/laminas/laminas-psr7bridge/blob/746ced1704a521058d8d5bb3ee205fd947344622/mkdocs.yml#L8

---

@froschdesign Does that mean it also has to be added here: https://github.com/laminas/laminas-psr7bridge/blob/1.3.x/docs/book/index.html or is this generated?

---

@setete
No, this page is automatically generated and uses the description from the file `mkdocs.yml`. Therefore this file must also be updated.